oozie-to-airflow/run-sys-test

```diff
@@ -341,7 +341,12 @@ function test_composer {
 }
 
 function prepare_configuration {
-    if [[ ! -f "${LOCAL_CONFIGURATION_PROPERTIES_TEMPLATE}" ]]; then
+    if [[ -f "${LOCAL_CONFIGURATION_PROPERTIES}" ]]; then
```

Comment:
I think we should always prepare local configuration - because otherwise overriding the values via command line parameters won't work and it will be misleading. There is no big harm in overwriting the configuration always, actually it is even good during development because then you will not forget to add it to the template (otherwise it will be continuously overwritten)

Reply:
You're right, I removed it.